client/src/dashboard/RewritingRuleModal.js

```diff
@@ -17,17 +17,21 @@ import defaultRulesData from '../mock-api/listRules';
 import { FormLabel } from '@mui/material';
 import RuleGraph from './RuleGraph';
 
-const AddRewritingRule = NiceModal.create(({user_id}) => {
+const RewritingRuleModal = NiceModal.create(({user_id, rule=null, queries=null}) => {
```

Comment:
If we remove the changes for `AddToMine`, we can remove the `queries` parameter.